src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunner.java

```diff
@@ -108,6 +116,22 @@
     return args;
   }
 
+  private String createFileForAuxClasspath(Set<File> auxClasspathFiles) {
+    String auxClasspath =
+        auxClasspathFiles.stream().map(File::getAbsolutePath).collect(Collectors.joining("\n"));
+    try {
+      File tempFile = File.createTempFile("temp", "-spotbugs-auxclasspath");
```

Comment:
I concern a little that it leads to fill in temp files in a local disk.
But, it can not fill soon (the file might be small).
It will be great If it can be persisted project-locally.

@KengoTODA 
What do you think?

Reply:
Hi @wreulicke , good point.

I changed this to write to the `build` folder instead: https://github.com/spotbugs/spotbugs-gradle-plugin/commit/2b9f18cf2adf2534ceeaff1e8e5df775cf8ba9a2